modules/space/shaders/fluxnodes_fs.glsl

```diff
@@ -0,0 +1,131 @@
+/*****************************************************************************************
+ *                                                                                       *
+ * OpenSpace                                                                             *
+ *                                                                                       *
+ * Copyright (c) 2014-2021                                                               *
+ *                                                                                       *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this  *
+ * software and associated documentation files (the "Software"), to deal in the Software *
+ * without restriction, including without limitation the rights to use, copy, modify,    *
+ * merge, publish, distribute, sublicense, and/or sell copies of the Software, and to    *
+ * permit persons to whom the Software is furnished to do so, subject to the following   *
+ * conditions:                                                                           *
+ *                                                                                       *
+ * The above copyright notice and this permission notice shall be included in all copies *
+ * or substantial portions of the Software.                                              *
+ *                                                                                       *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,   *
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A         *
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT    *
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF  *
+ * CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE  *
+ * OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                                         *
+ ****************************************************************************************/
+
+#include "fragment.glsl"
+
+in vec2 vs_st;
+in vec4 vs_color;
+in float vs_depth;
+in float vs_closeToEarth;
+in flat double vs_time;
+in float camera_IsCloseEnough;
+
+uniform sampler2D texture1;
+uniform bool drawCircles;
+uniform bool drawHollow;
+uniform bool useGaussian;
+uniform bool usingCameraPerspective;
+//uniform float testChange;
+uniform bool pulsatingAlways;
+uniform bool usingPulse;
+uniform bool usingGaussianPulse;
+uniform vec3 cameraPos;
+uniform vec4 streamColor;
+
+Fragment getFragment() {
+    vec4 fragColor = vs_color;
+    if (vs_color.a == 0) {
+        discard;
+    }
+   
+    vec2 pos = vec2(0.5)-vs_st;
```

Comment:
I'd suggest adding spaces around the operators in this and the next lines to make it easier to read